### PR TITLE
Update deprecated register methods

### PR DESCRIPTION
**Pull request by yux0**
Update deprecated register methods

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -114,18 +123,25 @@ github.com/uber-go/atomic v1.4.0 h1:yOuPqEq4ovnhEjpHmfFwsqBXDYbQeT6Nb0bwD6XnD5o=
...
 github.com/uber/tchannel-go v1.14.0 h1:v5mYnfCSI+H76umzo17+o3YdrnUt5W1AcvV+47065B0=
 github.com/uber/tchannel-go v1.14.0/go.mod h1:Rrgz1eL8kMjW/nEzZos0t+Heq0O4LhnUJVA32OvWKHo=
 go.uber.org/atomic v1.4.0 h1:cxzIVoETapQEqDhQu3QfnvXAV4AlzcvUCxkVUFw3+EU=
 go.uber.org/atomic v1.4.0/go.mod h1:gD2HeocX3+yG+ygLZcrzQJaqmWj9AIm7n08wl/qW/PE=
+go.uber.org/atomic v1.5.1 h1:rsqfU5vBkVknbhUGbAUwQKR2H4ItV8tjJ+6kJX4cxHM=
+go.uber.org/atomic v1.5.1/go.mod h1:sABNBOSYdrvTF6hTgEIbc7YasKWGhgEQZyfxyTvoXHQ=
 go.uber.org/cadence v0.8.9-0.20190711214419-cef99c5ba19e h1:OrD4zVun9PL0hAomg8y/JmkXs+Q9Y7+fZ5gtMx+G4eM=
 go.uber.org/cadence v0.8.9-0.20190711214419-cef99c5ba19e/go.mod h1:CQivfHCJ44B1kKL4LLtOhcepUwNoRodZceo/wU5Nthw=
+go.uber.org/cadence v0.13.4 h1:oqywBZx1NthaOqr+Ra92L7iWkwnXqqW/+napg2TKw7s=
+go.uber.org/cadence v0.13.4/go.mod h1:VQIAZxvehjwln/xaRv1ghzcxo7TjCHEetZQkc8NlrvE=
```
Comment (yycptt) [inline go.sum:144]: `go mod tidy`?

File: cmd/samples/fileprocessing/main.go [hunk 2]
```diff
@@ -47,6 +48,10 @@ func main() {
 
 	switch mode {
 	case "worker":
+		h.RegisterWorkflow(sampleFileProcessingWorkflow)
+		h.RegisterActivityWithAlias(downloadFileActivity, downloadFileActivityName, )
+		h.RegisterActivityWithAlias(processFileActivity, processFileActivityName, )
+		h.RegisterActivityWithAlias(uploadFileActivity, uploadFileActivityName, )
```
[the remark this answers is not in the document]
Comment (yux0) [reply cmd/samples/fileprocessing/main.go:54]: updated

File: go.sum (hunk 1 above)
Comment (yux0) [reply go.sum]: done

File: cmd/samples/recipes/branch/parallel_workflow.go [hunk 3]
```diff
@@ -11,14 +11,8 @@ import (
  * This sample workflow executes multiple branches in parallel using workflow.Go() method.
  */
 
-// This is registration process where you register all your workflows
-// and activity function handlers.
-func init() {
-	workflow.Register(SampleParallelWorkflow)
-}
-
 // SampleParallelWorkflow workflow decider
```
[the remark this answers is not in the document]
Comment (yux0) [reply cmd/samples/recipes/branch/parallel_workflow.go]: updated

**Pull request merged**